src/main/java/org/sagebionetworks/bridge/reporter/request/ReportScheduleName.java

```diff
@@ -0,0 +1,27 @@
+package org.sagebionetworks.bridge.reporter.request;
+
+import org.sagebionetworks.bridge.sqs.PollSqsWorkerBadRequestException;
+
+public enum ReportScheduleName {
+    DAILY("DAILY"),
+    WEEKLY("WEEKLY");
+
+    private final String name;
+
+    ReportScheduleName(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public String getSuffix() throws PollSqsWorkerBadRequestException {
+        if (this == DAILY) {
+            return "-daily-upload-report";
+        } else if (this == WEEKLY) {
+            return "-weekly-upload-report";
+        }
```

Comment:
This if-else is unnecessary. The suffix can be a property on the enum, like name. Or like how suffix was in the previous implementation.